lib/Instance/CustomVariable.js

```diff
@@ -270,6 +270,15 @@ export default class InstanceCustomVariable extends CoreBase {
 		}
 	}
 
+	getValue(name) {
+		if (this.custom_variables[name]) {
+			this.logger.silly(`Get value "${name}"`)
+			return this.custom_variables[name]
```

Comment:
I don't think this is correct for retrieving the value, this object describes the various options and details about the variable, everything except for its current value